### DISCUSSION TITLE
printer prematurely cooling down when printer is still printing

### DISCUSSION THREAD

**Discussion by agroupofpenguins** [Jyers/Marlin#796] [Q&A] [answered]
I'm having a little issue, when the print is almost done around 98% done, the print finish screen appears the confirm button appears and the printer starts to cooldown while still printing. I installed the firmware on the creality website and the issue was no longer there. I installed this firmware (default ender3 v2 4.2.2) and the issue was back. i tried previous releases of the firmware and the issue was present in those too. I'm new to 3d printing and epically changing the firmware, any help would be greatly appreciated. can add a video if needed.

**Comment by Jyers** [comment, marked as answer]
I think I know the cause of this and will try to fix it in the next release. I believe your gcode is sending a print finished command early, which causes the heaters to shut off. I'll see what I can do and feel free to update this is you still face the problem in the next release
